src/components/ModelTableList/StatusGroup/WarningMessage/WarningMessage.tsx

```diff
@@ -0,0 +1,69 @@
+import { List, Tooltip } from "@canonical/react-components";
+import type { ListItem } from "@canonical/react-components/dist/components/List/List";
+import { Link } from "react-router-dom";
+
+import ModelDetailsLink from "components/ModelDetailsLink";
+import type { ModelData } from "store/juju/types";
+import { getModelStatusGroupData } from "store/juju/utils/models";
+import urls from "urls";
+import { getUserName } from "utils";
+
+type Props = {
+  model: ModelData;
+};
+
+/**
+  Warning message for the model name cell.
+  @param model The full model data.
```

Comment:
This component doesn't have a `model` param anymore (the `model` arg is part of the props object). If you would like this comment to appear as a hint when using the component you could move this comment to the props type.

```typescript
type Props = {
  /**
   * The full model data.
   */
  model: ModelData;
};
```

Reply:
Fixed! Thanks for the review.